packages/js/react/src/provider.tsx

```diff
@@ -33,7 +33,7 @@ export function createWeb3ApiProvider(
     ClientContext: React.createContext({} as Web3ApiClient)
   };
 
-  return ({ redirects, plugins, interfaces, children }) => {
+  return ({ envs, redirects, plugins, interfaces, children }) => {
```

Comment:
Thank you for this. Is there a way to make this "future proof" so we don't have to keep updating this with new ClientConfig properties?